api/tests/test_game_state_server.py

```diff
@@ -90,3 +90,60 @@ def test_delete_after_load(gss_with_client):
         {'action': 'delete', 'data': valid_data['id']}, TEST_ROOM_ID,
     )
     assert len(reply.data) == 0
+
+
+def test_move_existing_token(gss_with_client):
+    gss_with_client.process_update(valid_update, TEST_ROOM_ID)
```

Comment:
I think it's worth testing a token update with at least one more token already in the store, to verify that updating a token doesn't affect other tokens